src/components/Drawer.jsx

```diff
@@ -416,18 +416,8 @@ const MobileDrawer = styled.div`
   align-content: center;
   width: 100vw;
   height: 9vh;
+  margin-bottom: 0vh;
 
-  ${(props) => props.open &&
```

Comment:
I think removing this bit of code actually impacts the functionality of the nav bar (it keeps header stuck at the top). 
Without this bit, it looks like:

https://user-images.githubusercontent.com/91639023/215820966-9dabe8d0-5f8b-45e9-a3b9-f58c68b19e4b.mp4

And with it it looks like:

https://user-images.githubusercontent.com/91639023/215821083-17f3a0f8-eb16-4db7-87f2-400da2d6c1cc.mp4